src/py/flwr/driver/driver.py

```diff
@@ -126,3 +128,77 @@ def pull_task_res(self, req: PullTaskResRequest) -> PullTaskResResponse:
         # Call Driver API
         res: PullTaskResResponse = self.stub.PullTaskRes(request=req)
         return res
+
+
+class Driver:
+    """`Driver` class provides an interface to the Driver API."""
+
+    def __init__(self) -> None:
+        self.grpc_driver = GrpcDriver()
+        self.workload_id = 0
```

Comment:
Let's make this optional to make the failure case more obvious:
```suggestion
        self.workload_id: Optional[int] = None
```